### PR TITLE
Decompile option menu

### PR DESCRIPTION
**Pull request by DizzyEggg**

### CODE REVIEW

File: src/option_menu.c [hunk 1]
```diff
@@ -0,0 +1,665 @@
...
+
+    styles[0] = 0;
+    styles[1] = 0;
+    styles[selection] = 1;
+
+    DrawOptionMenuChoice(gText_SoundMono, 104, 48, styles[0]);
+    DrawOptionMenuChoice(gText_SoundStereo, GetStringRightAlignXOffset(1, gText_SoundStereo, 198), 48, styles[1]);
+}
+
+// TODO: 19 should be defined in text_window.h
```
Comment (Diegoisawesome) [inline src/option_menu.c]: You should fix this

File: src/option_menu.c [hunk 2]
```diff
@@ -0,0 +1,665 @@
...
+    u8 n = selection + 1;
+    u16 i;
+
+    for (i = 0; gText_FrameTypeNumber[i] != EOS && i <= 5; i++)
+        text[i] = gText_FrameTypeNumber[i];
+
+    // Convert a number to decimal string
+    if (n / 10 != 0)
+    {
+        text[i] = n / 10 + CHAR_0;
```
Comment (Diegoisawesome) [inline src/option_menu.c]: Combine the text[i] and i++ lines

File: src/option_menu.c [hunk 3]
```diff
@@ -0,0 +1,665 @@
...
+    styles[selection] = 1;
+
+    DrawOptionMenuChoice(gText_ButtonTypeNormal, 104, 64, styles[0]);
+
+    widthNormal = GetStringWidth(1, gText_ButtonTypeNormal, 0);
+    widthLR = GetStringWidth(1, gText_ButtonTypeLR, 0);
+    widthLA = GetStringWidth(1, gText_ButtonTypeLEqualsA, 0);
+
+    widthLR -= 94;
+    yLR = (widthNormal - widthLR - widthLA) / 2 + 104;
```
Comment (Diegoisawesome) [inline src/option_menu.c]: Should be xLR

File: src/option_menu.c [hunk 4]
```diff
@@ -0,0 +1,665 @@
...
+        SetMainCallback2(MainCB2);
+        return;
+    }
+}
+
+static void Task_OptionMenuFadeIn(u8 taskId)
+{
+    if (!gPaletteFade.active)
+    {
+        gTasks[taskId].func = Task_OptionMenuProcessInput;
```
Comment (Diegoisawesome) [inline src/option_menu.c]: Remove brackets

File: src/option_menu.c (hunk 2 above)
Comment (DizzyEggg) [reply src/option_menu.c]: Won't match if I do that, it needs to stay the way it is.

**Pull request merged**